comp/README.md

```diff
@@ -131,3 +131,14 @@ Package systray implements the Datadog Agent Manager System Tray
 ### [comp/systray/systray](https://pkg.go.dev/github.com/DataDog/dd-agent-comp-experiments/comp/systray/systray)
 
 Package systray
+
+## [comp/trace](https://pkg.go.dev/github.com/DataDog/dd-agent-comp-experiments/comp/trace) (Component Bundle)
+
+*Datadog Team*: agent-apm
+
+Package process implements the "process" bundle, providing components for the Process Agent
```

Comment:
Should this have been changed? I think this should be talking about "Package trace" but talks about the process agent instead.